classes/class-woothemes-sensei-certificates.php

```diff
@@ -1090,13 +1099,13 @@ public function certificate_link( $message, $course_id = 0 ) {
 	/**
 	 * get_certificate_url gets url for certificate
 	 *
-	 * @access private
+	 * @access public
 	 * @since  1.0.0
 	 * @param  int $course_id course post id
 	 * @param  int $user_id   course learner user id
 	 * @return string $certificate_url certificate link
 	 */
-	private function get_certificate_url( $course_id, $user_id ) {
+	public function get_certificate_url( $course_id, $user_id ) {
```

Comment:
Do we really need to change this method to `public`? Checking the commits, I think it's maybe because it was being used by another class and it's not anymore?

Reply:
Oops! You're right! I'd changed it because it was being used [here](https://github.com/woocommerce/sensei-certificates/pull/271/commits/e5dbe8cc248e12fc4fc4b61e3d46ccba87ab1b13#diff-cd5b5d903e49a483e8ddac707897b0b0048e3a196b717b3e4c8631cb89cf1b3dR84), but then ended up deleting that file.

Anyways, it's fixed in https://github.com/woocommerce/sensei-certificates/pull/271/commits/9631705c1004c6401efd5ce8c29bf4f6693bb3c5. 🥂